### PR TITLE
[Embind] Fix getTypeName not defined

### PR DESCRIPTION
**Pull request by Minishlink**

### CODE REVIEW

**Comment by Minishlink** [comment]
Should I pull from the incoming branch ? My branch originates from master.

**Comment by waywardmonkeys** [comment]
Yes, branches need to go into ``incoming``.

File: src/embind/embind.js [hunk 1]
```diff
@@ -76,7 +76,7 @@ var LibraryEmbind = {
     }
     types.forEach(visit);
 
-    throw new UnboundTypeError(message + ': ' + unboundTypes.map(getTypeName).join([', ']));
+    throw new UnboundTypeError(message + ': ' + unboundTypes.map(___getTypeName).join([', ']));
```
Comment (waywardmonkeys) [inline src/embind/embind.js]: I don't think this is the right fix. I think the right fix is to add ``$getTypeName`` to the deps of ``$throwUnboundTypeError``

**Comment by Minishlink** [comment]
You may be completely right ! I didn't think it through and I just wanted to make a quick fix as I am not familiar at all with this.
I will review your suggestion, thank you. :)

**Comment by waywardmonkeys** [comment]
I'm pretty sure that the code as you have it now would just print the address of the type name rather than the actual string. :)

**Comment by Minishlink** [comment]
That can explain why I didn't understand the error haha ;)

**Comment by Minishlink** [comment]
In order to fix conflicts, should I fetch from incoming branch or will the project owner do the necessary steps when he will merge into master ?

**Comment by kripken** [comment]
I don't think github lets you change the merge target, so it would be easiest to make a new pull, rebased to incoming, and targeting incoming.

**Pull request closed without merging**